powdr_cli/src/main.rs

```diff
@@ -154,14 +155,20 @@ fn main() {
             );
         }
         Commands::RiscvAsm {
-            file,
+            files,
             inputs,
             output_directory,
             force,
         } => {
+            assert!(files.len() > 0);
+            let name = if files.len() == 1 {
```

Comment:
Why don't you make this a String? I don't see the benefit of Cow if you clone below.